modular_skyrat/modules/customization/modules/clothing/head/head.dm

```diff
@@ -336,3 +336,13 @@
 	greyscale_config_worn_teshari = /datum/greyscale_config/standalone_hood/worn/teshari
 	greyscale_config_worn_better_vox = /datum/greyscale_config/standalone_hood/worn/newvox
 	greyscale_config_worn_vox = /datum/greyscale_config/standalone_hood/worn/oldvox
+
+/obj/item/clothing/head/beret/badge
+	name = "badged beret"
+	desc = "The latest in Anti-Territorial Government fashion!"
```

Comment:
I don't really like what this is implying.

Reply:
The implication is that berets are the typical wear of rebels and anti-government militants.

I also mentioned the TG part in the body.

Reply:
I'm just not a fan of that being the description. If it was a comment, I wouldn't really say too much about it, but it's just the kind of very useless description that means absolutely nothing to everyone 3 months after it was added.

Reply:
Very well. It's a beret. Not much to describe it as though.